src/electionguard/precompute_buffers.cpp

```diff
@@ -204,14 +204,15 @@ namespace electionguard
         // This loop goes through until the queues are full but can be stopped
         // between generations of two triples and a quad. By full it means
         // we check how many quads are in the queue, to start with we will
-        // try 500 and see how that works. If the vendor wanted to pass the
+        // try 5000 and see how that works. If the vendor wanted to pass the
         // queue size in we could use that.
         // for now we just go through the loop once
         do {
             // TODO - Can we tolerate not returning until we have finished
             // generating two triples and a quadruple?
             // If not we can get more elaborate with the populate_OK checking
             std::lock_guard<std::mutex> lock(queue_lock);
+            int iteration_count = 0;
```

Comment:
This needs to be outside the do..while loop or it will always be 0

Reply:
Great catch Steve, that is a big oops.